src/LogContext/components/LogContextQueryBuilderSidebar.tsx

```diff
@@ -0,0 +1,191 @@
+import React, { useEffect, useMemo, useState } from "react";
+// import { Field } from '@grafana/data';
+import { useTheme2, CollapsableSection, Icon } from '@grafana/ui';
+import { LogContextProps } from "./LogContextUI";
+import { css, cx } from "@emotion/css";
+import { LuceneQuery } from "utils/lucene";
+import { LuceneQueryBuilder } from '@/QueryBuilder/lucene';
+
+
+// TODO : define sensible defaults here
+const excludedFields = [
+  '_source',
+  'sort',
+  'attributes',
```

Comment:
this is going to be an issue as this works only for otel indexes.
 
Can you open an issue on that?